manifests/s/Samsung/EasySettingBox/1.2.1/Samsung.EasySettingBox.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasySettingBox
+PackageVersion: 1.2.1
+Installers:
+- InstallerUrl: - https://downloadcenter.samsung.com/content/SW/202301/20230106083833260/EasySettingMSIX64.msi

```

Comment:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/202301/20230106083833260/EasySettingMSIX64.msi
```

Reply:
The esoteric error here [seems to refer](https://blogs.perl.org/users/brian_d_foy/2010/06/block-sequence-entries-are-not-allowed-in-this-context.html) to the data type implied by the en-dash.